docs/sites.yml

```diff
@@ -8150,3 +8150,27 @@
   source_url: https://github.com/IoT-Arduino/Gatsby-MySteps
   built_by: Maruo
   built_by_url: https://twitter.com/DengenT
+- title: Brent Runs Marathons
+  main_url: https://www.brentrunsmarathons.com/
+  url: https://www.brentrunsmarathons.com/
+  source_url: https://github.com/bingr001/brentrunsmarathonsv2
```

Comment:
This URL is pointing to a 404. Is the repo private or is this just an incorrect URL?

Reply:
```suggestion
```

Reply:
Whoops! it was. It's now public.